exonum/src/storage/proof_map_index/proof.rs

```diff
@@ -12,16 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// spell-checker:ignore lhash, rhash, lkey, rkey, bpath, repr
+// spell-checker:ignore bpath, deser, precheck, repr
```

Comment:
We already have "precheck" and "repr" in the "dictionary". Plus I prefer to replace `deser` by the full form.